.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+jdk:
+  - oraclejdk8
+script:
+  - gradle build
```

Comment:
Let's run `test` instead of `build`, as in addition to building the project it will also run the tests and fail if they don't pass. Because of the database dependencies we've set up, the build might not pass when we make this change. If that happens that's ok for this week, we can configure our project so that it passes properly next week.

You're also free to try and fix any failing tests own, but please don't do it on this branch as setting up CI and getting the CI build to pass are really different tasks, and should be done on different branches.

Reply:
Okay.